### ISSUE TITLE
Error message in depreciation report:  Whoops, looks like something went wrong

### ISSUE THREAD

**Issue by paulo1624** [snipe/snipe-it#4872]
#### Expected Behavior (or desired behavior if a feature request)

(what you expect to happen goes here)

-----

#### Actual Behavior

(what actually happens goes here)

-----

#### Please confirm you have done the following before posting your bug report:

- [ ] I have enabled debug mode 
- [ ] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

-----
#### Provide answers to these questions:

- Is this a fresh install or an upgrade? 
- Version of Snipe-IT you're running
- Version of PHP you're running
- Version of MySQL/MariaDB you're running
- What OS and web server you're running Snipe-IT on
- What method you used to install Snipe-IT (install.sh, manual installation, docker, etc)
- WITH DEBUG TURNED ON, if you're getting an error in your browser, include that error
- What specific Snipe-IT page you're on, and what specific element you're interacting with to trigger the error
- If a stacktrace is provided in the error, include that too.
- Any errors that appear in your browser's error console.
- Confirm whether the error is reproducible on the demo: https://snipeitapp.com/demo.
- Include any additional information you can find in `storage/logs` and your webserver's logs.
- Include what you've done so far in the installation, and if you got any error messages along the way.
- Indicate whether or not you've manually edited any data directly in the database

Please do not post an issue without answering the related questions above. If you have opened a different issue and already answered these questions, answer them again, once for every ticket. It will be next to impossible for us to help you.

https://snipe-it.readme.io/docs/getting-help
![image](https://user-images.githubusercontent.com/32555803/35201696-6d5b4dc6-ff58-11e7-9aed-2961e186789b.png)

**Comment by snipe** [comment]
Can you show me what's on line 79 of that view file?

**Comment by paulo1624** [comment]
Hi Snipe,

Please check the image below if this the correct file that you need. Thank You
![image](https://user-images.githubusercontent.com/32555803/35363467-12b32cdc-01a5-11e8-8b72-a2d526de0183.png)

**Comment by snipe** [comment]
I need to see line 79 of the file referenced. The one in `storage/framework/views`

**Comment by paulo1624** [comment]
This one?
![image](https://user-images.githubusercontent.com/32555803/35364583-ebaf4da4-01aa-11e8-9660-64803d3a0e86.png)

**Comment by snipe** [comment]
Is it possible the person an asset is assigned to was deleted?

**Comment by paulo1624** [comment]
I didn't do anything or deleting asset that assigned to users. I only did is to upgrade the version to 4.10 latest of snipeit, then I check all the function. I noticed that after clicking the depreciation report this error appear.

**Comment by chibacityblues** [comment]
@paulo1624 Same thing happend to us. @snipe is correct with her assumption. Here's what I did to identify the broken relationships: 

Replace the line:
`<?php echo e($asset->assigned->getFullNameAttribute()); ?>`
with this block:
```php
<?php
  try{
    echo e($asset->assigned->getFullNameAttribute());
  }
  catch(\Throwable $e){
    echo "<b>FUBAR</b>";
  }
?>
```

The report now should work and you can double check those assets with "FUBAR" in the "checked out" column.

**Comment by tehshane** [comment]
I encountered the same problem. It looks like the items _had_ been assigned to a now-deleted user, but were checked in before the user was removed (using the "Checkin & Delete User" button).

I looked in the DB, and while `assigned_to` was null, `assigned_type` was still set to `App\Models\User`. Once I nulled out that field in the affected assets (thanks @chibacityblues for the helpful troubleshooting idea!), the report started working again.
